### PR TITLE
Enables tracking.

### PR DESCRIPTION
**Pull request by ryansmith94**
This will require a new configuration value in the xapi-service.

### CODE REVIEW

**Comment by ryansmith94** [comment]
Note that the `batchSize` is set to the number of statements being stored, not the number of statements sent. This matches the functionality of V1.

**Comment by ryansmith94** [comment]
Note to self: Add sent batch size and client ID.

**Comment by ryansmith94** [comment]
```
process.env.NEW_RELIC_NO_CONFIG_FILE=true;
process.env.NEW_RELIC_APP_NAME=process.env.NEWRELIC_NAME;
process.env.NEW_RELIC_LOG=logDir + "/newrelic-agent.log"; ?????
process.env.NEW_RELIC_LOG_LEVEL="info";
const newrelic = require('newrelic');
```

File: src/service/storeStatements/index.ts [hunk 1]
```diff
@@ -66,6 +66,12 @@ export default (config: Config) => {
     await awaitUpdates(config, unawaitedUpdates);
     config.repo.emitNewStatements({ ids: statementIds });
 
+    (await config.tracker)('org_id', opts.client.organisation);
```
Comment (ht2) [inline src/service/storeStatements/index.ts]: Out of curiosity, why not store the result of one `(await config.tracker)` into a variable and then call on that var to make each call?

```
const tracker = await config.tracker;
tracker('org_id', opts.client.organisation);
tracker('lrs_id', opts.client.lrs_id);
tracker('batchSize', unstoredModels.length);
tracker('sentBatchSize', opts.models.length);
```

Comment (ryansmith94) [reply src/service/storeStatements/index.ts]: Yeah we certainly could.

Comment (ryansmith94) [reply src/service/storeStatements/index.ts]: I'll update that now.

Comment (ht2) [reply src/service/storeStatements/index.ts]: One other thing to check - do we need to `toString()` those IDs?

Comment (ryansmith94) [reply src/service/storeStatements/index.ts]: No they're strings already on the client, but good catch 👍

**Pull request merged**